packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -20,6 +19,11 @@ const double _kDragContainerExtentPercentage = 0.25;
 // displacement; max displacement = _kDragSizeFactorLimit * displacement.
 const double _kDragSizeFactorLimit = 1.5;
 
+/// The drag threshold after which the refresh indicator goes
+/// into [_RefreshIndicatorMode.armed].
```

Comment:
Nit: 
```suggestion
// The drag threshold after which the refresh indicator goes
// into [_RefreshIndicatorMode.armed].
```

I don't think these end up in docs since they are private